apps/pie-docs/src/_includes/header.njk

```diff
@@ -5,7 +5,7 @@
     {{ { name: 'close', attrs: { fill: toggleIconFill, height: '24', width: '24', class: 'u-iconFilled' } } | pieIconsSvg | safe }}
 </label>
 
-<header class="c-header" data-test-id="site_header">
+<header class="c-header" data-js="site-header" data-test-id="site_header">
```

Comment:
NIT: the data-js attribute in default-layout uses camel case. Should we stick to one approach?

Reply:
Updated all to kebab case